addon/manifests/chart/templates/apps.open-cluster-management.io_subscriptionstatuses_crd_v1alpha1.yaml

```diff
@@ -58,6 +58,14 @@ spec:
                 - lastUpdateTime
                 type: object
               type: array
+            subscription:
+              properties:
+                message:
+                  type: string
+                phase:
+                  description: PackagePhase defines the phasing of a Package
```

Comment:
description: PackagePhase defines the phasing of a Package
===>
 description: OverallPhase defines the overall phasing of a subscription

Reply:
The doc comes from the type:
`// PackagePhase defines the phasing of a Package
type PackagePhase string`

I'll define a new type called SubscriptionPhase, ok?

Reply:
ok